plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/nessie/IcebergNessieCatalogConfig.java

```diff
@@ -15,16 +15,33 @@
 
 import io.airlift.configuration.Config;
 import io.airlift.configuration.ConfigDescription;
+import io.airlift.configuration.ConfigSecuritySensitive;
+import io.airlift.units.Duration;
+import io.airlift.units.MinDuration;
 import jakarta.validation.constraints.NotEmpty;
 import jakarta.validation.constraints.NotNull;
+import org.projectnessie.client.NessieConfigConstants;
 
 import java.net.URI;
+import java.util.Optional;
+import java.util.concurrent.TimeUnit;
 
 public class IcebergNessieCatalogConfig
 {
+    public enum Security
+    {
+        NONE,
+        BEARER
+    }
+
     private String defaultReferenceName = "main";
     private String defaultWarehouseDir;
     private URI serverUri;
+    private Duration readTimeout = new Duration(NessieConfigConstants.DEFAULT_READ_TIMEOUT_MILLIS, TimeUnit.MILLISECONDS);
+    private Duration connectionTimeout = new Duration(NessieConfigConstants.DEFAULT_CONNECT_TIMEOUT_MILLIS, TimeUnit.MILLISECONDS);
```

Comment:
Static import `DEFAULT_READ_TIMEOUT_MILLIS`, `DEFAULT_CONNECT_TIMEOUT_MILLIS` and `MILLISECONDS`. 
Same for TestIcebergNessieCatalogConfig.